client/src/NetworkClient.ts

```diff
@@ -58,6 +71,17 @@ export class NetworkClient {
     private readonly _endpoint: string;
     private _eventClient!: EventClient;
     private $iframe!: HTMLIFrameElement;
+    private _apiLoadingState: 'not-started' | 'ready' | 'failed' = 'not-started';
+    private _consensus: 'syncing' | 'established' | 'lost' = 'syncing';
```

Comment:
Should be called `_consensusState` to not be confused with a reference to the consensus object.